include/nlohmann/detail/output/serializer.hpp

```diff
@@ -664,6 +659,20 @@ class serializer
         }
     }
 
+    /*!
+     * @brief convert a byte to a uppercase hex representation
+     * @param[in] byte byte to represent
+     * @return representation ("00".."FF")
+     */
+    inline std::string hex_bytes(std::uint8_t byte)
```

Comment:
Minor, especially since this is already merged: this is inside the class definition, so `inline` is implicit.  Also, this doesn't access any data, and so can be `static`.

Reply:
Thanks for noting. Fixed in 6d8d043adde8c95589f6e65e1fe228cb1f0d8419.